docs/concepts/serialization/types.md

```diff
@@ -214,7 +214,7 @@ Hex-encoded bytes serialized as a `u32` value describing the length of the bytes
 
 ## ByteCodeKind
 
-The type of byte code, serialized as a single `u8` value. A `0` indicates empty byte code, while a `1` indicates a `V1CasperWasm` to be executed with the first version of the Casper execution engine.
+The type of byte code, serialized as a single `u8` value. A `0` indicates empty byte code. `1` indicates a `V1CasperWasm` to be executed with the first version of the Casper execution engine. `2` indicates a `V2CasperWasm` to be executed with the first version of the Casper execution engine.
```

Comment:
Could you please confirm that there is no typo here regarding the "first version"?

"`V2CasperWasm` to be executed with the first version of the Casper execution engine."

Reply:
Sorry, per the code [here](https://github.com/casper-network/casper-node/blob/4c7068d44628757b9668ed3cac5e060e84ceef48/types/src/byte_code.rs#L250), it should be the second version. That's a typo, and I will fix it.

Reply:
Hi @mrkara , 
Fixed it now with the latest commit.